chapters/classes.tex

```diff
@@ -217,17 +217,27 @@ \subsubsection{Prefix Rules}\label{prefix-rules}
 
 A \firstuse{prefix} is property of an element of a class definition which can be present or not be present, e.g., \lstinline!final!, \lstinline!public!, \lstinline!flow!.
 
-Variables declared with the \lstinline!flow! or the \lstinline!stream! type prefix shall be a subtype of \lstinline!Real!.
-
 Type prefixes (that is, \lstinline!flow!, \lstinline!stream!, \lstinline!discrete!, \lstinline!parameter!, \lstinline!constant!, \lstinline!input!, \lstinline!output!) shall only be applied for type,
-record and connector components -- see also record specialized class, \cref{specialized-classes}.
+record, operator record, and connector components -- see also record specialized class, \cref{specialized-classes}.
```

Comment:
Remove line-break inside sentence.  (Unable to prepare suggestion.)

Reply:
Have done.